.github/workflows/build-reuse-darwin-framework.yml

```diff
@@ -48,8 +48,6 @@ jobs:
     steps:
     - name: Checkout repository
       uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9
-      with:
-        fetch-depth: 0
```

Comment:
I thought I had a reason for this...

Reply:
If a workflow needs to access tags, commit history or other branches than the ref branch, then it's needed.